lib/web_apis/version_history.es6.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  name: 'VersionHistory',
+  package: 'org.chromium.apis.web',
+  documentation: `VersionHistory is a class that stores official release
+    dates for each browsers and versions. It has methods to obtain
+    release date for a given browser and version.`,
+  properties: [
+    {
+      name: 'browserHistory',
+      documentation: `JSON of the form. :
+        {browserName: {versionNumber: releaseDate ...} ...}
+        The release dates for major browsers are collected manually
```

Comment:
Sounds like this is a singleton. Please add `axioms: [foam.pattern.Singleton.create()],` after `documentation` above.